tls/extensions/s2n_client_alpn.c

```diff
@@ -35,3 +35,62 @@ int s2n_extensions_client_alpn_send(struct s2n_connection *conn, struct s2n_stuf
 
     return 0;
 }
+
+int s2n_recv_client_alpn(struct s2n_connection *conn, struct s2n_stuffer *extension)
```

Comment:
/nit naming can be less confusing, what is `alpn` trying to convey here?

Reply:
this is a standard acronym for Application-Layer Protocol Negotiation as used in RFC.